packages/rich-text/src/plugins/Quote/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import * as Slate from 'slate-react';
+import { css } from 'emotion';
+import tokens from '@contentful/forma-36-tokens';
+import { EditorToolbarButton } from '@contentful/forma-36-react-components';
+import { Transforms, Editor, Node, Path } from 'slate';
+import { BLOCKS } from '@contentful/rich-text-types';
+import { useCustomEditor } from '../../hooks/useCustomEditor';
+import { CustomElement, CustomEditor } from 'types';
+
+const styles = {
+  blockquote: css({
+    margin: '0 0 1.3125rem',
+    borderLeft: `6px solid ${tokens.colorElementLight}`,
+    paddingLeft: '0.875rem',
+    fontStyle: 'normal',
+    '& a': {
+      color: 'inherit',
+    },
+  }),
+};
+
+interface ToolbarQuoteButtonProps {
+  isDisabled?: boolean;
+}
+
+const createBlockQuote = (editor: CustomEditor) => {
+  if (!editor.selection) return;
+
+  const text = { text: '' };
+  const paragraph = { type: BLOCKS.PARAGRAPH, children: [text] };
+  const path = editor.selection.focus.path;
+  const parent = Editor.parent(editor, path);
```

Comment:
I'm wondering if this is going to work if we start adding links to it, because links will add one more level so the parent won't be the parent you will be expecting, but this is just my theory.

You can also get the parent you want by using `Editor.levels` with the `reverse` property:

```js
const [currentListItemNode] = Editor.levels(editor, {
  at: editor.selection.focus.path,
  match: (node) =>
    Element.isElement(node) && (node as CustomElement).type === BLOCKS.LIST_ITEM,
  reverse: true,
});
```

I'm not sure if you will need it but we should test it later on. 

Can we add a TODO?

Reply:
Yeah I imagined this would break for links. I'll add a todo and link to this comment